plugin/core/typing.py

```diff
@@ -93,7 +97,8 @@ class Tuple(Type):  # type: ignore
     class Union(Type):  # type: ignore
         pass
 
-    Protocol = object  # type: ignore
+    class Protocol(Type):  # type: ignore
+        pass
```

Comment:
Changed this to make the `Protocol[T]` syntax not crash.
Those are just fake types so it doesn't matter what are those as long as they don't crash in Python <3.8.